backtrace-library/src/main/java/backtraceio/library/services/BacktraceReportSender.java

```diff
@@ -10,20 +10,27 @@
 import java.util.List;
 
 import backtraceio.library.common.BacktraceSerializeHelper;
+import backtraceio.library.common.BacktraceStringHelper;
 import backtraceio.library.common.MultiFormRequestHelper;
+import backtraceio.library.common.RequestHelper;
 import backtraceio.library.events.OnServerErrorEventListener;
 import backtraceio.library.logger.BacktraceLogger;
+import backtraceio.library.metrics.EventsPayload;
+import backtraceio.library.metrics.EventsResult;
 import backtraceio.library.models.BacktraceResult;
 import backtraceio.library.models.json.BacktraceReport;
+import backtraceio.library.models.types.BacktraceResultStatus;
 import backtraceio.library.models.types.HttpException;
 
 /**
  * Class for sending and processing HTTP request
  */
-class BacktraceReportSender {
+public class BacktraceReportSender {
```

Comment:
Are you sure it should be public? Shouldn't it stay internal? Is there any reason why user would like to use it?

Reply:
I made this public so I can use it from inside of `BacktraceEventsHandler` class. I will see if there's a way to avoid making this public.

Reply:
@konraddysput I can fix this by moving `metrics` source files to `services` package